holepunch/__init__.py

```diff
@@ -21,7 +21,9 @@
   -t --tcp               Open TCP ports to ingress [default].
   -u --udp               Open UDP ports to ingress.
   -y --yes               Don't prompt before writing rules.
-'''
```

Comment:
nit: can you please revert the unrelated change? i know `black` etc. would make this `"""`, but as is this would be inconsistent in the rest of the file